bot/exts/moderation/modpings.py

```diff
@@ -50,11 +50,17 @@ def __init__(self, bot: Bot):
             name="mod-pings-reschedule",
             event_loop=self.bot.loop,
         )
+        self.bot.loop.create_task(self.fetch_guild())
```

Comment:
Might be a nitpick, but this seems like a task that should come before the other two. Also notice that the task is still hanging on cog unload.

Reply:
I think the task can be hanging? It's not a looping task, it's one and done. Not entire sure how I would solve this.

Reply:
https://github.com/python-discord/bot/pull/2008/commits/188635730fb65bf8ebd6d7f1af154bfbf5b815ad